src/error/views/ErrorView.tsx

```diff
@@ -41,6 +47,10 @@ const ErrorView: FunctionComponent<ErrorViewProps> = ({
 			<Container size="medium" mode="horizontal">
 				<Blankslate body="" icon={errorIcon} title={errorMessage}>
 					{children}
+					<Button
```

Comment:
![image](https://user-images.githubusercontent.com/36598928/72972912-8d367300-3dcc-11ea-9609-5428d5422afe.png)

This button is now shown on every ErrorView, shouldn't happen on all.

Next to that, might be the perfect moment to remove that button "Zoek een fragment en maak je eerste collectie" as that was requested.

Reply:
> Next to that, might be the perfect moment to remove that button "Zoek een fragment en maak je eerste collectie" as that was requested.

waren ze er al uit of ze de knop weg wilden of dat het icoontje moest veranderen naar een "search" icon

Reply:
Geen idee.